build-system/tasks/changelog.js

```diff
@@ -536,7 +536,7 @@ function buildPrMetadata(pr) {
     id: pr.number,
     title: pr.title,
     body: pr.body,
-    merge_commit_sha: pr.merge_commit_sha,
+    merge_commit_sha: pr.merge_commit_sha,  // eslint-disable-line google-camelcase/google-camelcase
```

Comment:
This should use quotes to avoid.

Reply:
I tried quotes. The error is in `pr.merge_commit_sha`, which appears to be a well defined github constant.

Reply:
Both can use quotes.

Reply:
Just the field names needed quotes. The rest is fixed with `env` being set to `node`.